symbol.md

```diff
@@ -0,0 +1,181 @@
+# SYMBOLS#PROCS
+
+Thank you for accepting this mission, and the mission is nothing else but to add Symbols#procs to our Ruby toolchain . However, to achieve our mission, we need to have expertise of  **symbols** & **Procs** and then with their combined power we will take on the mission.
+
+If you have some difficulties with any of these concepts, do not worry, I made arrangements for you. [The Ruby_Newbie Guide to Symbols](http://www.troubleshooters.com/codecorn/ruby/symbols.htm), has exactly what you will need for symbols and [Ruby Explained: Blocks, Procs, and Lambdas, aka "Closures"](http://www.eriktrautman.com/posts/ruby-explained-blocks-procs-and-lambdas-aka-closures). Once you are well equipped, onward to the mission.
```

Comment:
I am not sure this statement is complete.
` I made arrangements for you. The Ruby_Newbie Guide to Symbols, has exactly what you will need for symbols and Ruby Explained: Blocks, Procs, and Lambdas, aka "Closures".`
Can you rephrase it?